source/org/zfin/search/Category.java

```diff
@@ -52,7 +52,7 @@ public enum Category {
             SOURCE,
             HOST_ORGANISM
     ),
-    ANATOMY("Anatomy / GO",
+    ANATOMY("Ontologies",
```

Comment:
Maybe change ANATOMY too

Reply:
Ok, I will change this when this in the main branch.